pages/become_a_member.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+    <div class="max-w-7xl mx-auto spx-6 px-6 lg:px-8">
+        <h1 class="text-2xl font-bold py-4">Become a Member</h1>
+        <p class="py-2">
+            Thank you for your interest in becoming a DES member! Firstly we would like to point out that <strong> you do not need to be a member to use the DES Slack, attend meetups and events, or to join the monthly society meeting and AGM. </strong> Membership is only required if you wish to vote, or use DES equipment and resources (like our YouTube channel or streaming tools.)
+        </p>
+        <p class="py-2">
+            If you're still interested in becoming a member, please complete the form below and then e-transfer $20 to memberships@devedmonton.com. Your membership will be valid for one year from the date that your membership was received. After the transfer is sent and you fill out the form you will be added to a google group which contains all our meetings and AGM's you can attend.
```

Comment:
I don't think this wording is in line with the by-laws. Membership is indefinite but can be revoked on non-payment of fees, and are not 1 year terms. @MarkBennett maybe you can chip in here as well. Here's my suggestion:

"If you're still interested in becoming a member, please complete the form below and e-transfer the $20 annual membership fees to memberships@devedmonton.com. On receipt of the completed form and membership fess, your application will be reviewed by the board. On approval of your application you will be added to a google group which contains all our meetings and AGM's you can attend."

Reply:
[The bylaws talk about fees and status in section 3.06.](https://docs.google.com/document/d/1i6oXabir-628csa2bQacHpc17Kmhec9EfkyHij9yQGM/edit#heading=h.vjzx5r62fvip) Here's the part that's probably relevant:

> 3.06 FEES
> The membership fee shall be an amount fixed from time to time by a majority vote of the Board, which vote shall become effective only when confirmed by a vote of the members at an annual or special general meeting.
>
> The Secretary shall notify the members (with the exception of honorary members) of the dues or fees at any time payable by them and, if any are not paid within 60 days of the date of such notice thereof, the members in default shall thereupon automatically cease to be members of the Society.
>
> Upon the failure of any member (with the exception of honorary members) to pay any membership fee, any subscription, or indebtedness due to the association, the Board may cause the name of such member to be removed from the register of members, but such member may be readmitted to membership by the Board upon such evidence as they may consider satisfactory.

I'd need to go back to our minutes for years past to find the motion, but I believe we voted on $20/year for a one year term. We also agreed at the last monthly meeting that members could pay us $20 via PayPal and we'd accept the fees PayPal charged us just to keep things simple.

Reply:
Right, I did say in my suggested wording that the fees are annual. I still don't think the original wording fits.

I also don't think that your statement (if you stop paying your fee then after a year you will no longer being a voting member) aligns with the bylaws. It's clear in the by-laws that membership isn't automatically revoked on non payment of fees, theres a process that needs to be followed to revoke membership (warning followed by 60 days grace period)